File: CSharp/k04_1.cs
using System;

namespace CSharp
{
    public class K04_1 : IExecutable
    {
        public void Execute()
        {
            try
            {
                Console.Write("年齢を入力してください。＞");
                int age = int.Parse(Console.ReadLine());

                if (age < 3 || age >= 70)
                {
                    Console.WriteLine("入場料金無料です。");
                }
                else
                {
                    Console.WriteLine("通常料金です。");
                }
            }
            catch (Exception ex)
            {
                Console.WriteLine(ex.Message);
            }

            return;
        }
    }
}

File: Python/k04_1.py
class k04_1(object):
    
    def execute(self):
        try:
            print('年齢を入力してください。＞')
            age = int(input())
            if age < 3 or age >= 70:
                print('入場料金無料です。')
            else:
                print('通常料金です。')
        except ValueError as ve:
            print(ve)